<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditWindow</class>
 <widget class="QMainWindow" name="EditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>810</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit data</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(250, 218, 221)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="tableView">
      <property name="maximumSize">
       <size>
        <width>622</width>
        <height>250</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 236, 236);
font: 63 15pt &quot;Yrsa&quot;;
color: rgb(176, 0, 0);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(250, 218, 221);
font: 63 15pt &quot;Yrsa&quot;;
color: rgb(176, 0, 0);</string>
      </property>
      <property name="title">
       <string>Edit data</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font: 63 25pt &quot;Yrsa&quot;;
color: rgb(176, 0, 0);</string>
         </property>
         <property name="text">
          <string>&amp;Name</string>
         </property>
         <property name="buddy">
          <cstring>NameLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="NameLineEdit">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
font: 63 25pt &quot;Yrsa&quot;;
color: rgb(176, 0, 0);</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font: 63 25pt &quot;Yrsa&quot;;
color: rgb(176, 0, 0);</string>
         </property>
         <property name="text">
          <string>&amp;Surname</string>
         </property>
         <property name="buddy">
          <cstring>SurnameLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="SurnameLineEdit">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
font: 63 25pt &quot;Yrsa&quot;;
color: rgb(176, 0, 0);</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">font: 63 25pt &quot;Yrsa&quot;;
color: rgb(176, 0, 0);</string>
         </property>
         <property name="text">
          <string>&amp;Groups</string>
         </property>
         <property name="buddy">
          <cstring>GroupsComboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="GroupsComboBox">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
font: 63 25pt &quot;Yrsa&quot;;
color: rgb(176, 0, 0);</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">font: 63 25pt &quot;Yrsa&quot;;
color: rgb(176, 0, 0);</string>
         </property>
         <property name="text">
          <string>&amp;Language</string>
         </property>
         <property name="buddy">
          <cstring>LangComboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="LangComboBox">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
font: 63 25pt &quot;Yrsa&quot;;
color: rgb(176, 0, 0);</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true">font: 63 25pt &quot;Yrsa&quot;;
color: rgb(176, 0, 0);</string>
         </property>
         <property name="text">
          <string>&amp;Level</string>
         </property>
         <property name="buddy">
          <cstring>LevelComboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="LevelComboBox">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
font: 63 25pt &quot;Yrsa&quot;;
color: rgb(176, 0, 0);</string>
         </property>
         <item>
          <property name="text">
           <string>Beginner</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Pre-Intermediate</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Intermediate</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Upper Intermediate</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Advanced</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Proficiency</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true">font: 63 25pt &quot;Yrsa&quot;;
color: rgb(176, 0, 0);</string>
         </property>
         <property name="text">
          <string>&amp;Teacher</string>
         </property>
         <property name="buddy">
          <cstring>TeacherComboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="TeacherComboBox">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
font: 63 25pt &quot;Yrsa&quot;;
color: rgb(176, 0, 0);</string>
         </property>
         <item>
          <property name="text">
           <string>Native</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Non-Native</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_7">
         <property name="styleSheet">
          <string notr="true">font: 63 25pt &quot;Yrsa&quot;;
color: rgb(176, 0, 0);</string>
         </property>
         <property name="text">
          <string>&amp;Price</string>
         </property>
         <property name="buddy">
          <cstring>PriceLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="PriceLineEdit">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
font: 63 25pt &quot;Yrsa&quot;;
color: rgb(176, 0, 0);</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="SaveButton">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(250, 218, 221);
font: 63 25pt &quot;Yrsa&quot;;
color: rgb(176, 0, 0);</string>
        </property>
        <property name="text">
         <string>&amp;Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="DeleteButton">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(250, 218, 221);
font: 63 25pt &quot;Yrsa&quot;;
color: rgb(176, 0, 0);</string>
        </property>
        <property name="text">
         <string>&amp;Delete</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>639</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>NameLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>EditWindow</receiver>
   <slot>SaveButtonSetEnabled()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>16</x>
     <y>542</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SurnameLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>EditWindow</receiver>
   <slot>SaveButtonSetEnabled()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>353</y>
    </hint>
    <hint type="destinationlabel">
     <x>62</x>
     <y>564</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PriceLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>EditWindow</receiver>
   <slot>SaveButtonSetEnabled()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>508</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>560</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SaveButtonSetEnabled()</slot>
  <slot>TableViewClicked()</slot>
 </slots>
</ui>
